src/Voron/Constants.cs

```diff
@@ -46,10 +47,13 @@ public static class Size
         public const int CurrentVersion = 7;
 
         public const string RootTreeName = "$Root";
+        public static readonly Slice RootTreeNameSlice = Slice.From(StorageEnvironment.LabelsContext, RootTreeName, ByteStringType.Immutable);
```

Comment:
how thread safe is this? If we have two threads initializing two different static fields using the same label context?